mobile-app/app/screens/AppNavigator/screens/Portfolio/components/BottomSheetAddressDetailV2.tsx

```diff
@@ -162,7 +162,7 @@ export const BottomSheetAddressDetailV2 = (props: BottomSheetAddressDetailProps)
     return (
       <ThemedTouchableOpacityV2
         key={item}
-        style={tailwind('p-4 flex flex-row items-center justify-between border-0 mx-5 rounded-lg-v2 h-20')}
+        style={tailwind('p-5 flex flex-row items-center justify-between border-0 mx-5 rounded-lg-v2 h-20')}
```

Comment:
```suggestion
        style={tailwind('px-5 py-4.5 flex flex-row items-center justify-between border-0 mx-5 rounded-lg-v2 h-20')}
```

Reply:
@kyleleow, isn't it 20px on the Figma file?

Reply:
measure from the text, since the text has a taller height